build.gradle

```diff
@@ -124,6 +121,8 @@ sourceSets {
     satori {
         java {
             srcDirs 'src/satori/java'
+            srcDirs += 'build/generated/source/proto/satori/java'
```

Comment:
I think good to keep this because it's explicit, but just FYI I think the proto plugin was doing this implicitly.

Reply:
You're right I think it was, it's just that IntelliJ was not resolving symbols without these lines.